src/plugins/discover/public/application/doc_views/doc_views_types.ts

```diff
@@ -32,7 +32,9 @@ import { ComponentType } from 'react';
 import { SearchResponse } from 'elasticsearch';
 import { IndexPattern } from '../../../../data/public';
 
-export type OpenSearchSearchHit<T = unknown> = SearchResponse<T>['hits']['hits'][number];
+export type OpenSearchSearchHit<T = unknown> = SearchResponse<T>['hits']['hits'][number] & {
```

Comment:
nit: the name `OpenSearchSearchHit` is confusing for me because it seems like OpenSearch is returning back to us if it is an anchor or not. Can we create another alias for this to be more descriptive that shims to the hit response if it isAnchor or not based on the context?

that would help me as well not worry if there is potential issues changing the type (even though it is just an optional field). if this type is actually only used within Discover in context then perhaps the name can be reconsidered since `OpenSearchSearchHit` implies to me it has more purposes than just for table rows.